src/Raven.Server/ServerWide/Commands/ETL/UpdateEtlProcessStateCommand.cs

```diff
@@ -20,6 +20,8 @@ public class UpdateEtlProcessStateCommand : UpdateValueForDatabaseCommand
 
         public string NodeTag { get; set; }
 
+        public string DbId { get; set; }
```

Comment:
for old commands this can be null when deserialized, also consider mixed cluster where one node is e.g. 5.2.2 and a second one is 5.2.3